wgsl/index.bs

```diff
@@ -4644,10 +4643,6 @@ See [[#builtin-inputs-outputs]] for how to declare a built-in variable.
          [[WebGPU#shader-output-mask|shader-output mask]].
          Zero bits in the written value will cause corresponding samples in
          the color attachments to be discarded.
-         <br>The value in the variable is significant only if the
```

Comment:
This reminds me that the shader-output mask description in the WebGPU spec needs updating as well.  It talks about statically-used.  It also uses the SV_Coverage terminology from HLSL.  I'll file an issue separately for that.

Reply:
oh, I thought I removed those statically used thingies (for I/O). Missed this one, thank you for spotting!